### PR TITLE
#275 - extracted repo settings to enum

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #275 I've extracted repository settings to enum to avoid duplications and added instruction to continue #275.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#276](https://github.com/artipie/rpm-adapter/issues/276) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/6d6af587-65be-42bb-b87a-4fa36c5becb9); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/rpm/RpmOptions.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        this.name = name;
+        this.arg = opt;
+        this.desc = desc;
+    }
+
+    /**
+     * Builds command line option.
+     * @return Instance of {@link Option}.
+     */
+    public Option option() {
```
Comment (Vatavuk) [inline src/main/java/com/artipie/rpm/RpmOptions.java:98]: @olenagerasimova make this method final just to be sure

File: src/main/java/com/artipie/rpm/RpmOptions.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+ *  settings:
+ *  --digest: sha256
+ *  --naming-policy: sha1
+ *  --filelists: true
+ *  as described in https://github.com/artipie/artipie/issues/227. Consider extracting interface
+ *  from CliParsedArguments, the interface may have two implementation: FromCliArguments and
+ *  FromYaml. After that these settings should be passed to `RpmSlice` and `RpmUpload` to create
+ *  `Rpm` instance accordingly to the settings.
+ */
+public enum RpmOptions {
```
[the remark this answers is not in the document]
Comment (olenagerasimova) [reply src/main/java/com/artipie/rpm/RpmOptions.java:41]: @Vatavuk it's not only rpm cli options, it's more like rpm repository configuration options. These same options will be used on storage settings level, see https://github.com/artipie/artipie/issues/227

**Comment by olenagerasimova** [comment]
@Vatavuk thanks, corrected and answered

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality good

**Pull request merged**